Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -104,16 +106,14 @@ def Runner(env):
                 args += f" -drive file=\"{path_to_os}\",format={storage_format},if=none,id=os_disk"
                 args += " -device ahci,id=ahci"
                 args += " -device ide-hd,drive=os_disk,bus=ahci.0"
-        elif os.path.isfile(VirtualDrive):
-            args += f" -drive file={VirtualDrive},if=virtio"
-        elif os.path.isdir(VirtualDrive):
-            args += f" -drive file=fat:rw:{VirtualDrive},format=raw,media=disk"
         else:
-            logging.critical("Virtual Drive Path Invalid")
-
-        # TODO: Set the memory size to be 2GB regardless. Not sure why 8GB does
-        # not work.
-        args += " -m 2048"
+            args += " -m 2048"

```

Comment:
Isn't this still setting 2048 always? it would just set 8192 first and then 2048 after?